content/java/concepts/math-methods/terms/subtract-exact/subtract-exact.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.subtractExact()'
+Description: 'Returns the difference of the arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.subtractExact()`** method returns the difference of the arguments.
+
+## Syntax
+
+```pseudo
+Math.subtractExact(x, y)
+```
+
+Datatype of the value returned by '.subtractExact()' method depends upon the arguments passed:
+
+- If both the arguments are of type `int` then the return type will be `int`.
+- If both the arguments are of type `long` then the return type will be `long`.
+- If one of the arguments is of `long` type then the return will be `long`.
+
+## Example
+
+The following example demonstrates the `.subtractExact()` method being applied to two variables:
```

Comment:
"The following example demonstrates the `.subtractExact()` method: ~being applied to two variables:~"